core/vm/jump_table.go

```diff
@@ -67,6 +67,7 @@ type JumpTable [256]*operation
 // constantinople, istanbul, petersburg, espresso and g-fork instructions.
 func newGforkInstructionSet() JumpTable {
 	instructionSet := newEspressoInstructionSet()
+	enable3529GFork(&instructionSet) // EIP-3529: Reduction in refunds https://eips.ethereum.org/EIPS/eip-3529
```

Comment:
Hmm, shouldn't it be `enable3529`?

Reply:
There's `enable3529` in `newEspressoInstructionSet`, but here the dynamic gas calculation function changes, so we need a different name. 

I could also rename other to `enable3529Celo` or similar if that makes it clearer. Let me know what you think.

Reply:
Yeah, I think renaming makes sense but also I mean `enable3529GFork` uses `gForkGasSStoreEIP3529` which in turn uses `params.CeloSstoreClearsScheduleRefundEIP3529` which equals to `6100`, not `4800` or I'm missing something?

Reply:
Yes, good catch, I mixed those up.